src/pages/accountSetup/AccountSetupPage.jsx

```diff
@@ -0,0 +1,378 @@
+                      />
+                      {/* <StyledFormHelperText id="main-text-point-helper-text">
+                          {formik.touched.mainTextPointSize &&
+                            formik.errors.mainTextPointSize}
+                        </StyledFormHelperText> */}
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <Typography variant="subtitle1" sx={{ fontSize: 20 }}>
+                        Your System Name
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <FormControl
+                        fullWidth
+                        error={
+                          formik.touched.systemName &&
+                          Boolean(formik.errors.systemName)
+                        }
+                      >
+                        <CustomTextField
+                          placeholder="Enter"
+                          aria-describedby="system-name-helper-text"
+                          fieldName={"systemName"}
+                          formik={formik}
+                        ></CustomTextField>
+
+                        <StyledFormHelperText id="system-name-helper-text">
+                          {formik.touched.systemName &&
+                            formik.errors.systemName}
+                        </StyledFormHelperText>
+                      </FormControl>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <Typography variant="subtitle1" sx={{ fontSize: 20 }}>
+                        Two-Factor Authentication
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <StyledCheckbox
+                        name="twoFactAuth"
+                        // do not remove double negation or else your see an uncontrolled to controlled switchBase error
+                        // refer to https://stackoverflow.com/questions/69259429/material-ui-a-component-is-changing-the-uncontrolled-checked-state-of-switchbas
+                        checked={!!formik.values.twoFactAuth}
+                        onChange={formik.handleChange}
+                      />
+                    </Grid>
+                  </Grid>
+                  <Box height={30} />
+                  <Box
+                    sx={{
+                      display: "flex",
+                      float: "right",
+                      justifyContent: "space-between",
+                      width: "30%",
+                    }}
+                  >
+                    <Button
+                      color="primary"
+                      variant="contained"
+                      type="reset"
+                      sx={{
+                        padding: 0,
+                        ":hover": { backgroundColor: "transparent" },
+                      }}
+                      disabled={isSubmitting}
```

Comment:
This can probably use the use the "navigate(-1)" feature to go back
See https://www.datainfinities.com/36/create-a-back-button-with-react-router